articles/140_topic_and_service_name_mapping.md

```diff
@@ -76,14 +73,15 @@ The content of substitutions:
 
 - must not be empty
 
+- must be not longer than 256 characters
```

Comment:
That is only partially correct. The size of the RTPS topic name is limited to 256 characters and thus the ROS base name is limited to 256.
I'll update that.